### ISSUE TITLE
PHP 8.1 compatibility

### ISSUE THREAD

**Issue by LeBenLeBen** [mzur/kirby-form#14]
When using the library with PHP 8.1, it triggers the following deprecation notice:

```
htmlspecialchars_decode(): Passing null to parameter #1 ($string) of type string is deprecated
```

**Comment by mzur** [comment]
This should have been fixed by #9 wich was released with v3.0.1. Which version do you use?

**Comment by LeBenLeBen** [comment]
Oh my, yes indeed, I overlooked the latest major (I’m using it through uniform). Please forgive me for the useless noise.
